File: src/main/org/nlogo/prim/etc/_clearoutput.java
package org.nlogo.prim.etc;

import org.nlogo.nvm.Command;
import org.nlogo.nvm.Context;
import org.nlogo.nvm.Syntax;

public final strictfp class _clearoutput
    extends Command {
  @Override
  public Syntax syntax() {
    return Syntax.commandSyntax();
  }

  @Override
  public void perform(final Context context) {
    workspace.clearOutput();
    context.ip = next;
  }
}

File: src/main/org/nlogo/prim/etc/_clearoutput.scala
// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo

package org.nlogo.prim.etc

import org.nlogo.api.Syntax
import org.nlogo.nvm.{ Command, Context }

class _clearoutput extends Command {
  override def syntax =
    Syntax.commandSyntax
  override def perform(context: Context) {
    workspace.clearOutput()
    context.ip = next
  }
}
